internal/cluster/transition.go

```diff
@@ -542,7 +551,23 @@ func (th *transitionHandler) PostPreparingTimedOut(sw stateswitch.StateSwitch, a
 		return err
 	}
 
-	eventgen.SendInstallationPreparingTimedOutEvent(params.ctx, params.eventHandler, *cluster.ID)
+	allHostsPrepared := th.areAllHostsPreparedForInstallation(cluster)
```

Comment:
I thought that hosts are handling their timeout.  Why are we handling this here ?